ipatests/test_integration/test_ipa_cert_fix.py

```diff
@@ -278,52 +292,86 @@ def install(cls, mh):
             setup_dns=False, extra_args=['--no-ntp']
         )
 
-    def test_renew_expired_cert_replica(self):
+    @pytest.fixture
+    def move_date_check_replication(self):
+        # add user to check replication after renewal
+        testuser = 'testuser1'
+        password = 'Secret@123'
+
+        # move system date to expire certs
+        for host in self.master, self.replicas[0]:
+            move_date(host, 'stop', '+3years+1days')
+
+        yield
+
+        # check user on replica
+        stdin = (f"{self.master.config.admin_password}\n"
+                 f"{self.master.config.admin_password}\n"
+                 f"{self.master.config.admin_password}\n")
+        self.master.run_command(['kinit', 'admin'], stdin_text=stdin)
+        tasks.user_add(self.master, testuser, password=password)
+        self.replicas[0].run_command(['kinit', 'admin'], stdin_text=stdin)
+        self.replicas[0].run_command(['ipa', 'user-show', testuser])
```

Comment:
What is the point of the above lines? Do you want to check that replication is working after the certs have been renewed? In this case the test belongs to the method test_renew_expired_cert_replica rather than to the fixture.